rskj-core/src/jmh/java/co/rsk/jmh/web3/BenchmarkWeb3.java

```diff
@@ -240,6 +240,31 @@ public void traceGet(BasePlan plan) throws BenchmarkWeb3Exception {
         plan.getWeb3Connector().traceGet(transactionHash, Stream.of("0x0").collect(Collectors.toList()));
     }
 
+
+    @Benchmark
+    @Timeout(time = 30)
```

Comment:
do we really need (here and below) this custom timeout?

Reply:
not really, removing them.